specs/testing/skills/SkillsFunctionalTesting.md

```diff
@@ -109,13 +109,14 @@ Given these elements, the number of test cases for each scenario can be calculat
 
 **Total test cases:** 96 (not including alternate flows)
 
-### 3. Skill sends a proactive message to consumer
+### 3. Skill sends a proactive message to consumer - conversation update
 
 > A consumer calls a _timer skill_ and the skill finishes the conversation but stores some tasks. Then, some time later the skill sends an update to the consumer as a message.
 
+
 **Testing matrix**
 
-- Skill: TBD
+- Skill: [Proactive messaging](https://github.com/microsoft/BotBuilder-Samples/tree/main/samples/csharp_dotnetcore/16.proactive-messages)
```

Comment:
The sample won't work as it is in for the scenario, it will need to be refactored as a skill. 

I think it may be better if we:
1 Create a #### Proactive skill placeholder in the ### Skills section of the doc at the bottom and there we describe that the skill is based on the 16-proactive-messages samples (with the link to the samples repo).
2 In this scenario, just add a ref to the #### Proactive skill section at the bottom of the doc

Reply:
moved to bottom of spec